tests/test_container.py

```diff
@@ -88,6 +88,48 @@ class Something(object):
     Something([1, 2, 3, 4])
 
 
+@pytest.mark.parametrize('test_type, correct, wrongs', [
+    (str, 'foo', [42, True]),
+    (int, 42, ['foo', ()]),
+])
+def test_typing_newtype_single(test_type, correct, wrongs):
+    SomeNew = typing.NewType('SomeNew', test_type)
+
+    validator = type_validator(True)
```

Comment:
empty_ok already is set to True as default so you don't need to provide it here.

Reply:
Explicit is better then implicit.

Reply:
I agree, but I don't think that applies to default arguments. What's the reason for default arguments if we're going to pass the value in like there's no default ?

Reply:
Defaults are a recommendation and a nice to the user if he does not care about some values. However, this not an user applications, but tests and in a single unit tes we want to explicitly test a specific case, without implicit assumptions (except for an own test which should test if defaults working correct). That way we explicit sey what we want to test.